test/dx_mgn_pool.js

```diff
@@ -27,10 +27,40 @@ contract("DxMgnPool", (accounts) => {
       assert.equal(await instance.numberOfParticipations.call(accounts[0]), 1)
       const participation = await instance.participationAtIndex.call(accounts[0], 0)
       assert.equal(participation[1], 10)
-
+      
       assert.equal(await instance.totalDeposit.call(), 10)
       assert.equal(await instance.totalPoolShares.call(), 10)
     })
+    it("checks right participationIndex", async () => {
```

Comment:
this would be more clear if it was something like "first participation index is always even" (or something like this.